packages-web/maps/webpack.config.prod.js

```diff
@@ -1,7 +1,28 @@
 const merge = require("webpack-merge");
 const baseConfig = require("../utils-react-widgets/configs/webpack.config.prod");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
 
 const customConfig = {
+    output: {
+        filename: "widgets/com/mendix/widget/custom/Maps/Maps.js"
+    },
+    module: {
+        rules: [
+            {
+                test: /\.(png|jpg|gif|svg|eot|ttf|woff|woff2)$/,
+                loader: "url-loader"
+            }
+        ]
+    },
+    plugins: [
```

Comment:
This is already in the webpack config of utils-react-widgets, so I think it's not necessary here.

Reply:
This is the solution for the missing CSS and the Typo in Maps ID